src/css/custom.css

```diff
@@ -154,6 +154,10 @@ body {
 	background-color: var(--background);
 }
 
+.alert a {
+	text-decoration: var(--ifm-link-decoration);
+}
```

Comment:
@cody-dot-js I added this because I noticed that the links in alerts (info/warning boxes) were not looking like links and just blended into the text. This at least underlines them now.